wsmaster/che-core-api-factory-azure-devops/src/main/java/org/eclipse/che/api/factory/server/azure/devops/AzureDevOpsURLParser.java

```diff
@@ -71,10 +71,6 @@ public AzureDevOpsUrl parse(String url) {
     }
 
     String repoName = matcher.group("repoName");
-    if (repoName.endsWith(".git")) {
-      repoName = repoName.substring(0, repoName.length() - 4);
-    }
```

Comment:
@tolusha do you happen to remember why this check was added?
I suspect that it was needed for smth. but I might be wrong

Reply:
Copy/paste from [1] 
[1] https://github.com/eclipse-che/che-server/blob/df3fc9965aab47fb958513d0545ee1531fd6e0ae/wsmaster/che-core-api-factory-gitlab/src/main/java/org/eclipse/che/api/factory/server/gitlab/GitlabUrlParser.java#L167